utils/doclint/documentation.js

```diff
@@ -802,12 +802,12 @@ function generateSourceCodeComment(spec) {
       node.codeLang = parseCodeLang(node.codeLang).highlighter;
     if (node.type === 'note') {
       // @ts-ignore
-      node.type = 'text';
-      node.text = '**NOTE** ' + node.text;
+      node.type = 'text'
+      node.text = `**NOTE** ` + (node.children || []).map(child => md.render([child], { flattenText: false, omitLastCR: true })).join('↵')
```

Comment:
I'd recommend adding a new rendering option `renderNoteAs: '**NOTE** '` instead, and remove special handling of notes in this method entirely.